flox/nn/typing.py

```diff
@@ -1,4 +1,8 @@
+from typing import TypeAlias, Literal
```

Comment:
should keep the typing import format consistent

Reply:
Noted, will correct.